File: 백준/Bronze/5235. Even Sum More Than Odd Sum/Even Sum More Than Odd Sum.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    n,*arr=[*map(int,input().split())]
    total=[0]*2
    for c in arr:
        total[c%2]+=c
    print("TIE" if total[0]==total[1] else "EVEN" if total[0]>total[1] else "ODD")

File: 백준/Bronze/5235. Even Sum More Than Odd Sum/Even Sum More Than Odd Sum.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int k, n, elem;
    cin >> k;
    for (int i = 0; i < k; i++) {
        cin >> n;
        vector<int> total(2);
        for (int j = 0; j < n; j++) {
            cin >> elem;
            total[elem % 2] += elem;
        }
        cout << (total[0] == total[1] ? "TIE" : (total[0] > total[1] ? "EVEN" : "ODD")) << "\n";
    }
    return 0;
}